staging/provisioning/windows/windowsnodereset.ps1

```diff
@@ -15,6 +15,7 @@ $global:NetworkMode = "L2Bridge"
 $global:NetworkPlugin = $Global:ClusterConfiguration.Cni.Name
 $global:ContainerRuntime = $Global:ClusterConfiguration.Cri.Name
 $UseContainerD = ($global:ContainerRuntime -eq "containerd")
+$IsDualStackEnabled = $Global:ClusterConfiguration.Kubernetes.Kubeproxy.FeatureGates -contains "IPv6DualStack=true"
```

Comment:
Not a blocker for this PR but we should also think about how we would do this in `v1.21+`. Dual-stack is beta as of `v1.21` and these feature gates are enabled by default. So they will not be explicitly set.